init/cli/lib.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2016 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"io/ioutil"
+	"os"
+	"syscall"
+
+	"github.com/palantir/pkg/cli"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	"github.com/palantir/go-java-launcher/launchlib"
+)
+
+const (
+	outputFileFlag       = os.O_CREATE | os.O_WRONLY
+	truncOutputFileFlag  = outputFileFlag | os.O_TRUNC
+	appendOutputFileFlag = outputFileFlag | os.O_APPEND
+	outputFileMode       = 0666
```

Comment:
This should probably be 644, no reason allowing everyone write access to the output file